live-examples/js-examples/string/string-endswith.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var str1 = 'Cats are the best';
+
+console.log(str1.endsWith('best'));
+// expected output: true
+
+var str2 = 'Is this a question';
+
+console.log(str2.endsWith('?'));
+// expected output: false
+</code>
```

Comment:
Do you think it is worth showing the `length` parameter too, using something like:

```
console.log(str1.endsWith('best', 16));
// expected output: false
```